tests/tests_async.py

```diff
@@ -94,3 +94,10 @@ def test_async_call_with_defaults(self):
             lambda_function_name="MyLambda",
         )
         lambda_async_mock.return_value.send.assert_called_with(get_func_task_path(async_me), ("qux",), {})
+
+    def test_async_call_arg_not_json_seralizable(self):
```

Comment:
seralizable -> ser**i**alizable
minor typo :) just had to double check myself when typing this word as well :)

(tagging the comment on this line, but there are 7 occurences in this PR)

Reply:
Fixed spelling error.

Reply:
@hellno updated.